statsmodels/survey/survey_model.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+
+class SurveyModel(object):
+
+    def __init__(self, design, model_class, init_args={}, fit_args={}):
+        self.design = design
+        self.model = model_class
+        self.init_args = dict(init_args)
+        self.fit_args = dict(fit_args)
+
+    def _centering(self, array=None):
```

Comment:
`array` is a mandatory argument to this function, better not to give it a default.  Also `_center` would be better name than `_centering` (function names should usually be verbs).